htmltest/check-link_test.go

```diff
@@ -369,13 +369,39 @@ func TestMailtoBlank(t *testing.T) {
 	tExpectIssue(t, hT, "mailto is empty", 1)
 }
 
-func TestMailtoInvalid(t *testing.T) {
+func TestMailtoInvalidFormat(t *testing.T) {
 	// fails for invalid mailto links
 	hT := tTestFile("fixtures/links/invalid_mailto_link.html")
 	tExpectIssueCount(t, hT, 1)
 	tExpectIssue(t, hT, "contains an invalid email address", 1)
 }
 
+func TestMailtoInvalidCname(t *testing.T) {
+	// fails for invalid mailto links
+	hT := tTestFile("fixtures/links/invalid_mailto_cname.html")
+	tExpectIssueCount(t, hT, 0)
```

Comment:
Guessing this isn't done yet. In general write the tests with expected result and add `t.Skip("Not yet dealt with")` at the top.

Reply:
The test was complete, it was checking that just a CNAME pointing to another domain that was valid would work as expected, comments should hopefully clear that up.